modules/oe_corporate_countries_address/tests/src/Kernel/AvailableCountriesSubscriberTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_corporate_countries_address\Kernel;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\Tests\oe_corporate_countries\Kernel\CorporateCountriesRdfKernelTestBase;
+
+/**
+ * Tests the AvailableCountriesSubscriber class.
+ */
+class AvailableCountriesSubscriberTest extends CorporateCountriesRdfKernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'user',
+    'entity_test',
+    'address',
+    'oe_corporate_countries_address',
+  ];
+
+  /**
+   * Tests the deprecated countries are removed from the countries array.
+   */
+  public function testGetAvailableCountriesArray(): void {
+
+    $entityTypeName = 'entity_test';
+
+    FieldStorageConfig::create([
+      'field_name' => 'field_country',
+      'entity_type' => $entityTypeName,
+      'type' => 'address_country',
+    ])->save();
```

Comment:
You're creating a field storage that is not being used in your field config created below, so what is this for?

Reply:
Hello Alexandra, 
We need this initializations for this line of code later

```
    $actual_countries = $entity->get('field_country')
      ->appendItem()
      ->getAvailableCountries();
```

If we do not use this code we get:

```
InvalidArgumentException: Field field_country is unknown.
```

Reply:
This has not been removed in the last push.